docs/source/key_components/rl_toolkit.rst

```diff
@@ -58,138 +54,115 @@ easily turned into a roll-out worker or an actor for synchronous and asynchronou
 Policy
 ------
 
-The ``AbsPolicy`` abstraction is the core component of MARO's RL toolkit. A policy is a an agent's mechanism to select
-actions in its interaction with an environment. MARO allows arbitrary agent-to-policy mapping to make policy sharing
-easily configurable. It is also possible to use a mixture of rule-based policies and ``RLPolicy`` instances. The latter
-provides various policy improvement interfaces to support single-threaded and distributed learning.   
-
-
-.. code-block:: python
-
-  class AbsPolicy(ABC):
-      def __init__(self, name)
-          super().__init__()
-          self._name = name
-
-      @abstractmethod
-      def __call__(self, state):
-          """Select an action based on a state."""
-          raise NotImplementedError
+``Policy`` is the most important concept in reinforcement learning. In MARO, the highest level abstraction of a policy
+object is ``AbsPolicy``. It defines the interface ``get_actions()`` which takes a batch of states a inputs and returns
```

Comment:
a inputs -> as input (I believe input should be singular here, but feel free to double check)